processing/functions/swaps.js

```diff
@@ -166,12 +167,39 @@ const module = {
     const totalAmount = transactionAmount - totalFee;
 
     return {
-      swaps,
+      swaps: validSwaps,
       totalAmount,
       totalFee,
     };
   },
 
+  /**
+   * Filter out any swaps which are deemed as invalid.
+   *
+   * For BLOKI_TO_LOKI, a swap is invalid if:
+   *  The total amount from an address is less than the fee
+   *
+   * @param {[{ uuid, amount, address }]} swaps The swaps
+   * @param {string} swapType The swap type
+   * @returns the filtered out swaps
+   */
+  filterInvalidSwaps(swaps, swapType) {
```

Comment:
it sounds like it returns only invalid swaps, maybe filterOutInvalidSwaps? or filterValidSwaps?

Reply:
Ah yes, now that i see it, it doesn't make much sense. 👍 i'll change it